library/core/src/convert/num.rs

```diff
@@ -172,7 +172,8 @@ impl_from! { f32, f64, #[stable(feature = "lossless_float_conv", since = "1.6.0"
 #[stable(feature = "float_from_bool", since = "1.68.0")]
 #[rustc_const_unstable(feature = "const_num_from_num", issue = "87852")]
 impl const From<bool> for f32 {
-    /// Converts `bool` to `f32` losslessly.
+    /// Converts `bool` to `f32` losslessly. The resulting value is `0.0` for
```

Comment:
We can be a little more specific, since floats have signed zeros.
```suggestion
    /// Converts `bool` to `f32` losslessly. The resulting value is `+0.0` for
```

This should also be repeated on `f64` below.